packages/squiggle-lang/benchmark/generators.ts

```diff
@@ -0,0 +1,48 @@
+export let generateFloatRange = (min: number, max: number): number =>
```

Comment:
any reason why you didn't use the fast-check types? I think they have a bunch of helpers for things like this.